src/pages/Components/Notification/code.md

```diff
@@ -7,26 +7,36 @@ order: 2
 
 ```css
 @import 'settings-tools/_all-settings';
-@import '...';
+@import 'components/_c.notification';
 ```
 
 ## Basic usage
 
-Apply the classes...
+Wrap a h3, p and a a or button tags in a div and apply the `mc-notification` class:
 
 ```html
-<div class="class">Label</div>
+  <div class="mc-notification">
+    <h3>Notification title</h3>
+    <p>
+      Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa.
+    </p>
+    <a href="#">Link</a>
+  </div>
 ```
 
-<preview path="src/pages/Components/Notification/Notification/previews/Notification"></preview>
+<preview path="src/pages/Components/Notification/previews/notification-danger"></preview>
 
-## Variations
+## States
 
-use the same variations in the same order as the index page
+You have 4 states of notification available (self-explanatory): *information*(default), *success*, *warning* and *danger*. Apply them using the corresponding class: `mc-notification-[type]`.
```

Comment:
maybe put the states in bullet points like we're doing with other patterns. 

![image](https://user-images.githubusercontent.com/5315842/64544504-6a2c8a80-d327-11e9-9d61-a90b9d26be28.png)


![image](https://user-images.githubusercontent.com/5315842/64544538-77e21000-d327-11e9-8837-d88c0e0c790e.png)